demo/node-typescript/src/index.ts

```diff
@@ -8,10 +8,130 @@ console.log('Running with api token ' + api_token);
 
 // Create the Strapi client instance
 const client = strapi({ baseURL: 'http://localhost:1337/api', auth: api_token });
-// Create a collection type query manager for the categories
-const categories = client.collection('categories');
 
-// Fetch the list of all categories
-const docs = await categories.find();
+async function runDemo() {
```

Comment:
What do you think about keeping a smaller example for files (e.g. finding files related to the found categories) so that it can easily be maintainer and replicated across all demo projects?

Reply:
I'm not sure how that would work - we'd have to use a different content type
that has files associated e.g. authors and query the files by the contents of `files_related_mph` in the
DB? I guess by using `await client.files.find({ filters: { ... } });`

Reply:
We could also modify the Strapi app content-types so that categories have an image associated with them and then just query the categories + their image?